src/chart/pie/PieView.js

```diff
@@ -319,7 +327,8 @@ var PieView = ChartView.extend({
         );
 
         var selectedMode = seriesModel.get('selectedMode');
-
+        var valueDim = data.mapDimension('value');
+        data._sum = data.getSum(valueDim) || 0;
```

Comment:
By convention, a property name prefixed by a single `'_'` represents the property is a "private member" of an class. We should not add other that kind of properties (`_xxx`) outside the class  definition dynamically. Otherwise, that probably confuse the code reader and might overwrite the inner private members with the same name. 

If we need save the `sum` here, it's OK to to save it to `this`, because in this module we have the legal control of `this`.